app/helpers/feature_flag.rb

```diff
@@ -1,7 +1,15 @@
 module FeatureFlag
   extend self # rubocop:disable Style/ModuleFunction
 
+  def strict_enabled?(feature_name, *args)
+    Flipper.enabled?(feature_name, *args)
+  end
+
+  def exist?(feature_name)
+    Flipper.exist?(feature_name)
+  end
+
   def enabled?(feature_name, *args)
-    Flipper[feature_name].enabled?(*args)
+    !exist?(feature_name) || strict_enabled?(feature_name, *args)
```

Comment:
The default behavior of Flipper is to return false is a flag isn't defined. I've added this helper method to return true if:

feature flag is globally enabled
feature flag is disabled, but enabled for the specified user id
feature flag has not been defined
I need input on this last point: if you were to call FeatureFlag.enabled? for a flag that didn't exist...what would you expect to get?

Reply:
Oh I see, that check totally went over my head. 

From that perspective I'd maybe say it could be good to make `FeatureFlag.enabled?` return `false` for a flag that doesn't exist.

My reasoning is that if someone sets a check for a feature_flag with a typo then the page will leak as generally available (regardless if @citizen428 idea of using the slug is implemented or not).

Reply:
Hmm, that's a fair point. I think I'm covered with this logic for `FeatureFlag.accessible?`

feature flag is globally enabled: **returns true**
feature flag is disabled, but enabled for the specified user id: **returns true**
feature flag specified doesn't exist: **returns false**
nil or empty string passed in: **returns true**